<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Авторизация</string>
    </property>
    <widget class="QLabel" name="labelLogin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AuthButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>войти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toRegButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>к регистрации</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLogin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="RegButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>зарегистрироваться</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toAuthButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>к авторизации</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrepID">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID преподавателя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPrepID">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxStatus">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Преподаватель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Студент</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelFio">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>    ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEye">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../server_beta/release/img/witness.png</normaloff>../server_beta/release/img/witness.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
